src/ubuntu/manifest.json

```
@@ -638,7 +638,7 @@
         {
           "platforms": [
             {
-              "dockerfile": "src/ubuntu/22.04/amd64",
+              "dockerfile": "src/ubuntu/22.04/",
```

Nit: just for consistency

```suggestion
              "dockerfile": "src/ubuntu/22.04",
```

---

Sure thing, thanks for noticing, fixed